### ISSUE TITLE
Unmounting <Stage> prints warning

### ISSUE THREAD

**Issue by Xaaq** [inlet/react-pixi#251]
### Description
When trying to conditionally render `<Stage>` component the first unmounting operation causes following warning to be printed in the console:
```
Warning: React instrumentation encountered an error: Error: Expected root pseudo key to be known. 
```

Mount operations do not cause any warnings - only unmounts do that. Warning only appears on the first unmount operation - subsequent unmounts don't print any warnings.

### Steps to reproduce
1. Add conditional rendering of `<Stage>` component. Minimal example:
    ```
    import React, {useState} from 'react';
    import ReactDOM from 'react-dom';
    import {Stage} from "@inlet/react-pixi";

    function App() {
        const [doShow, setDoShow] = useState(true)
    
        return <>
            {doShow && <Stage></Stage>}
            <button onClick={() => setDoShow(!doShow)}>
                SHOW / HIDE
            </button>
        </>
    }
    
    ReactDOM.render(<App/>, document.getElementById("root"))
    ```
2. Click on `SHOW / HIDE` button to unmount `<Stage>` component.
3. See warning that printed in the console.

### Additional info
- `@inlet/react-pixi` version:  `5.1.5`
- `React` version: `16.13.1`
- `ReactDOM` version: `16.13.1`
- `PIXI` version: `5.3.3`

**Comment by inlet** [comment]
Hi @Xaaq thanks for letting me know! I've just pushed an update and I'm happy to inform you that this is now fixed in `v5.1.7`. It turns out that `injectIntoDevTools` needs to be called just once and as you can see the devtools is now correctly in sync:

![ScreenFlow](https://user-images.githubusercontent.com/232559/95681685-975e0300-0be1-11eb-822a-0165b0c77193.gif)

**Comment by Xaaq** [comment]
Thank you very much!
